### ISSUE TITLE
Small doc fix for linux install

### ISSUE THREAD

**Issue by g4borg** [kgraefe/pidgin-hide-chat-on-join#11]
Building it in linux also needs package "intltool" which is not installed by default for build-essential.

**Comment by kgraefe** [comment]
But it should be installed with the `pidgin-dev` which is what the README.md actually says.  Can you confirm this? What is your distribution?

**Comment by g4borg** [comment]
```
$ apt-cache rdepends pidgin-dev
pidgin-dev
Reverse Depends:
```

```
$ apt-cache depends pidgin-dev
pidgin-dev
  Depends: pidgin
  Depends: pidgin
  Depends: pkg-config
    pkg-config:i386
    pkgconf:i386
    pkgconf
  Depends: libpurple-dev
  Depends: libgtk2.0-dev
  Suggests: doc-base
```

i have neon, which builds on ubuntu 18.04; 
the pidgin packages are untouched by the neon crew, so it is basicly the same as vanilla buntu.
i had pidgin-dev installed.

```
$ apt-cache policy pidgin-dev
pidgin-dev:
  Installed: 1:2.12.0-1ubuntu4
  Candidate: 1:2.12.0-1ubuntu4
  Version table:
 *** 1:2.12.0-1ubuntu4 500
```

**Comment by g4borg** [comment]
doing a more in depth dependency i am amazed myself, why it was not present, as if i do a recursive dependency list it clearly shows intltool is there - but it is hard to follow the dependency link that far down

however uninstalling pidgin-dev intltool
and only reinstalling pidgin-dev, does not install intltool.

it is rather probaly installed by gnome. i do not have gnome.

**Comment by kgraefe** [comment]
Using `apt-rdepends pidgin-dev | grep intltool` it seems that `pidgin-dev` really does not depend on `intltool`. It may be that Pidgin can be built without translation support but my plugins require it.
